health/health.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package health
+
+import (
+	"sync"
+	"time"
+
+	"github.com/vechain/thor/v2/thor"
+)
+
+type BlockIngestion struct {
+	BestBlock          *thor.Bytes32 `json:"bestBlock"`
+	BestBlockTimestamp *time.Time    `json:"bestBlockTimestamp"`
```

Comment:
This could be misunderstood by the best block's timestamps itself, while this timestamp is from when it gets processed. Maybe `BestBlockTimestamp` can be renamed to `BestBlockIngestionTime` or `...ProcessedTime`

Reply:
Good point - fixed to `BestBlockIngestionTimestamp`